packages/app-admin/src/components/OverlayLayout/OverlayLayout.tsx

```diff
@@ -13,7 +13,7 @@ const OverlayLayoutWrapper = styled("div")({
     width: "100%",
     height: "100vh",
     backgroundColor: "var(--mdc-theme-background)",
-    zIndex: 9, // has to be higher than 5 so it's above advanced settings dialog, and below 10, so the image editor can be displayed above
+    zIndex: 50, // has to be higher than 5 so it's above advanced settings dialog, and below 10, so the image editor can be displayed above
```

Comment:
@TommyJackson85 one thing to check here: please open the file manager, and then try editing an image, see if image editor is displayed correctly. The comment says `...and below 10, so the image editor can be displayed above`. If image editor is not being displayed, please increase its zIndex as well.